core/templates/dev/head/css/oppia.css

```diff
@@ -2144,3 +2150,126 @@ md-card.preview-conversation-skin-supplemental-card {
 .trained-feedback-selection:hover {
   color: #00897B;
 }
+
+.oppia-modal-information-card .modal-dialog {
+  margin: 100px auto;
+  width: 400px;
+}
+
+.oppia-modal-information-card .modal-dialog .modal-content {
+  border-radius: 0;
+}
+
+.oppia-modal-information-card .modal-dialog .modal-content md-card {
+  border-radius:0;
+  margin: 0;
+  padding: 0;
+}
+
+.oppia-information-card-title {
+  bottom: 0;
+  color: white;
+  font-family: "Capriola", "Roboto", Arial, sans-serif;
+  margin: 12px 0;
+  padding: 0;
+  position:absolute;
+}
+
+.oppia-information-card-logo-thumbnail {
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 300px;
+  padding: 20px;
+  position: relative; 
+}
+
+.oppia-information-card-content {
+  padding: 10px 20px 10px 20px;
+}
+
+.oppia-information-card-content p {
+  display: block;
+  font-size: 1.0em;
+  line-height: 1;
+  padding: 12px 0 24px 0;
+  position: relative;
+  word-spacing: 0; 
+}
+
+.oppia-information-card-content .card-metrics {
+  margin-top: 10px;
+  padding: 0;
+}
+
+.oppia-information-card-content .card-metrics li {
+  font-size: 14px;
+  list-style-type: none;
+  text-align: center;
+}
+
+.oppia-information-card-content .card-metrics ul img {
+  height: 24px;
+  margin-right: 2px;
+  width: 24px;
+}
+
+.oppia-contributors-more-circle {
+  background: #888;
+  border-radius: 50%;
+  border-color: black;
+  height: 24px;
+  padding-top: 3px;
+  text-align: center; 
+  width: 24px;
+}
+
+.oppia-information-card-content .card-metrics li .fa {
+  font-size: 24px;
+  margin-right: 5px;
+}
+
+.oppia-social-media-and-tags {
+  margin-top: 20px
+}
+
+.oppia-social-media-and-tags md-chips md-chip {
+  font-size: 12px;
+  line-height: 1em;
+  margin: 0;
+  padding: 0;
+}
+
+.oppia-information-card-tooltip-more {
+  color: blue;
+}
+
+/* This is needed to overide the maximum width of tooltip
+   otherwise  breaks when you have longer text in tooltip */
+.tooltip-inner {
```

Comment:
I looked into this. Here's what it looked like for me *without* the tooltip-inner class:

![without-tag](https://cloud.githubusercontent.com/assets/10575562/10329472/dacccfdc-6c76-11e5-9fbd-f17d1d5fd012.png)

I think this is fine, actually, and thus I think it's also fine to remove the .tooltip-inner.

---

On a different note, could you please add a single space immediately after every comma that separates two tags? It's more readable that way.

In addition, try entering a small tag followed by a long one. The tooltip section starts to look a bit weird. Perhaps the truncation logic needs to be adjusted (or alternatively we can also say, it's probably OK for now, since long tags like this are unlikely anyway):

![toomanytags](https://cloud.githubusercontent.com/assets/10575562/10329481/0105a2d2-6c77-11e5-8e3e-e4363ddf8723.png)